mappings/net/minecraft/block/Block.mapping

```diff
@@ -135,6 +135,7 @@ CLASS none/akw net/minecraft/block/Block
 	METHOD a getRawIdFromBlock (Lnone/akw;)I
 		ARG 0 block
 	METHOD a (Lnone/akw;Lnone/akw;)Z
+	METHOD a setSounds (Lnone/apu;)Lnone/akw;
```

Comment:
Depends what the class itself is called. If the class is called Sounds setSounds is fine, if the class is call SoundType then it should be setSoundType